GX-AnalyticContinuation/ABINIT/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+cmake_minimum_required(VERSION 3.16)
```

Comment:
Is this required?

I wouldn't distinguish this directory as ABINIT. The routines should be as code agnostic as possible